pkg/webhook/inject.go

```diff
@@ -55,7 +57,7 @@ func (v *PodInjector) Handle(ctx context.Context, req admission.Request) admissi
 	log.Info("Get request from pod:", "pod", pod)
 
 	return admission.Response{
-		AdmissionResponse: *inject.Inject(&req.AdmissionRequest, v.client, v.Config, v.ControllerCfg, v.Metrics),
+		AdmissionResponse: *i.Inject(&req.AdmissionRequest, v.client, v.Config, v.ControllerCfg, v.Metrics),
```

Comment:
do not use global value `i`.

consider calling the constructor to instant new `Injector` instance.

 you could modify the fields of `webhook.PodInjector`, appending new field `logger`, so you could access logger when you instant new `Injector` instance.